files/galaxy-test/tpv/users.yml

```diff
@@ -1,22 +1,4 @@
 ---
 users:
   kuntzm@informatik.uni-freiburg.de:
```

Comment:
Does tpv dry-run complain if `kuntzm@informatik.uni-freiburg.de` is empty? Could you quickly check?

Reply:
It does not complain